kernel/README.md

```diff
@@ -0,0 +1,79 @@
+## Kernel images
+
+The builder provides firecracker-compatible images for 2 kernel versions,
+`5.10.77` and `4.19.215`.
```

Comment:
Do we have to include specific versions here? I have a feeling when we bump up the kernel version, even just a patch version for example because of a security patch, we will forget this readme ever existed and the version number will be the same forever. Maybe `5.10` and `4.19` is enough (?).

Reply:
I think we just have to note somewhere, like a PR template or release process with a checkbox "[ ] Updated all related docs (this, that, and that as well)"

Reply:
yep and yep, will make those changes